apps/meteor/tests/end-to-end/api/00-miscellaneous.js

```diff
@@ -509,6 +510,19 @@ describe('miscellaneous', function () {
 					done();
 				});
 		});
+		it('create a team', (done) => {
```

Comment:
Make it async

Reply:
Also, this is not asserting anything, maybe it's better for a "before" statement?

Reply:
Agreed! Thankss